third_party/WebKit/Source/modules/webcl/WebCLHTMLUtil.h

```diff
@@ -6,7 +6,10 @@
 #ifndef WebCLHTMLUtil_h
 #define WebCLHTMLUtil_h
 
-#include <wtf/RefCounted.h>
+#include "platform/graphics/Image.h"
+#include "platform/graphics/ImageBuffer.h"
```

Comment:
Isn't forward declaration enough in this header file? I prefer to put headers in .cpp files if possible.